src/main/static/src/components/common/henkilo/HenkiloViewContactContent.js

```diff
@@ -120,10 +121,11 @@ class HenkiloViewContactContent extends React.Component<Props, State> {
     };
 
     render() {
-
+        
         const passivoitu = this.props.henkilo.henkilo.passivoitu;
         const duplicate = this.props.henkilo.henkilo.duplicate;
-        const hasHenkiloReadUpdateRights: boolean = hasAnyPalveluRooli(this.props.omattiedot.organisaatiot, ['OPPIJANUMEROREKISTERI_HENKILON_RU', 'OPPIJANUMEROREKISTERI_REKISTERINPITAJA']);
+        const hasHenkiloReadUpdateRights: boolean = this.props.view === 'OMATTIEDOT' ? true : hasAnyPalveluRooli(this.props.omattiedot.organisaatiot, ['OPPIJANUMEROREKISTERI_HENKILON_RU', 'OPPIJANUMEROREKISTERI_REKISTERINPITAJA']);
```

Comment:
this.props.view === 'OMATTIEDOT' || hasAnyPalveluRole() voisi olla hieman selkeämpi.